src/sass/_common/reskin.scss

```diff
@@ -48,7 +48,12 @@ body {
     padding-block: 32px;
 }
 
-
+// Hide persistent chart showing in under the main chart
+div[data-highcharts-chart="0"]:not(.quill-purchase-popup div[data-highcharts-chart="0"]) {
+    position: absolute;
+    top: -9999px;
```

Comment:
`display:none` won't work because there's script applying `display:block` directly to the element, `opacity` is not an option too cause it will occupy a space below the main chart.